frontend/src/host_orchestrator/orchestrator/instancemanager_test.go

```diff
@@ -42,6 +42,7 @@ func (AlwaysSucceedsValidator) Validate() error {
 }
 
 const fakeLatesGreenBuildID = "9551522"
+const cvdUser = "_cvd-executor"
```

Comment:
`fakeCVDUser` for consistency with other names.

Reply:
Done